cranelift/codegen/src/isa/zkasm/inst/mod.rs

```diff
@@ -434,9 +433,15 @@ fn zkasm_get_operands<F: Fn(VReg) -> VReg>(inst: &Inst, collector: &mut OperandC
                     .expect("Only support 1 register in comparison now"),
                 b0(),
             );
+            if cc == IntCC::NotEqual {
+                // zkasm don't have NEQ opcode so we need a bit more computation
+                let mut clobbered = PRegSet::empty();
+                clobbered.add(c0().to_real_reg().unwrap().into());
+                clobbered.add(d0().to_real_reg().unwrap().into());
+                collector.reg_clobbers(clobbered);
```

Comment:
These registers aren’t getting clobbered though?

Reply:
And this, of course, is a mistake, sorry =( [Fixed](https://github.com/near/wasmtime/pull/57/commits/8f76d7f0bdd69990831be833f798ac532cbea83a)